contrib/stack/stripmapStack/trackCRs.py

```diff
@@ -104,7 +103,7 @@ def getAzRg(frame,llh):
     pol._normRange = frame.instrument.rangePixelSize
     pol.initPoly(azimuthOrder=0, rangeOrder=len(coeffs)-1, coeffs=[coeffs])
 
-    taz, rgm = frame.orbit.geo2rdr(list(llh), side=frame.instrument.platform.pointingDirection, 
+    taz, rgm = frame.orbit.geo2rdr(list(llh)[1:], side=frame.instrument.platform.pointingDirection,
```

Comment:
Looks like this essentially changes the format of the text file that the script expects, essentially one more column. Can we add a simple comment indicating the format that the script accepts.

Reply:
Yeah, these changes were needed for the CSVs providing Rosamond CR data. Is that format atypical?

Reply:
Yeah. Its a simple text format - non-standardized. I think it would be good to add what is expected.